src/Command/Accounts/AddSshKeyCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Pagely\AtomicClient\Command\Accounts;
+
+use GuzzleHttp\Exception\ClientException;
+use Pagely\AtomicClient\API\Accounts\AccountsClient;
+use Pagely\AtomicClient\API\AuthApi;
+use Pagely\AtomicClient\Command\Command;
+use Pagely\AtomicClient\Command\OauthCommandTrait;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AddSshKeyCommand extends Command
+{
+    use OauthCommandTrait;
+
+    /**
+     * @var AccountsClient
+     */
+    protected $api;
+
+    public function __construct(AuthApi $authApi, AccountsClient $apps, $name = 'account:ssh:add')
+    {
+        $this->authClient = $authApi;
+        $this->api = $apps;
+        parent::__construct($name);
+    }
+
+    public function configure()
+    {
+        parent::configure();
+        $this
+            ->setDescription('Add a ssh key for an account - feed ssh key in via stdin')
+            ->addUsage('Feed ssh key to command via stdin. Example: cat /path/to/my.key | ./bin/atomic account:ssh:add 37')
+            ->addArgument('accountId', InputArgument::REQUIRED, 'Account ID')
+        ;
+        $this->addOauthOptions();
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        $accountId = $input->getArgument('accountId');
+        $token = $this->token->token;
+        $key = '';
+        while (!feof(STDIN)) {
+            $key .= fread(STDIN, 1024);
+        }
+
+        if (empty(trim($key))) {
+            $output->writeln('No key given');
+            return 1;
+        }
+
+        try {
+            $this->api->addSshKey($token, $accountId, $key);
```

Comment:
I'm assuming we have this method in place somewhere else. We are validating params on mgmt side right?